index.html

```diff
@@ -162,216 +162,95 @@
-                         <div class="col-xs-12 col-sm-3">
-                        
-                        </div>
-                         <div class="col-xs-12 col-sm-3">
-                            <ul class="sources sources2">
-                                <li class="lightgrey">Javascript</li>
+        <!--***************** END SECOND COLUMN *******************-->
 
-                                
-                                <i class="fa fa-star inline lightgrey"></i>
-                                <li class="lightgrey inline">9</li>
-                                
 
 
-                                
-                                <i class="fa fa-code-fork inline lightgrey"></i>
-                                <li class="lightgrey inline"> 3</li>
-                                
-                               
-                            </ul>
-                        </div>
-                        
-                    </div>
-                    <p>Babel is a compiler for writing next generation JavaScript.</p>
-                    <p class="lightgrey">Updated a day ago</p>
-                    <div class="row">
-                        <div class="col-sm-6"></div>
-                        <div class="col-sm-5">
-                            <div class="bar bar1"></div>
-                            <div class="bar bar2"></div>
-                            <div class="bar bar3"></div>
-                            <div class="bar bar4"></div>
-                            <div class="bar bar5"></div>
-                            <div class="bar bar6"></div>
-                            <div class="bar bar7"></div>
-                            <div class="bar bar8"></div>
-                            <div class="bar bar9"></div>
-                           
-                        </div>
-                        <div class="col-sm-1"></div>
-                    </div>
-                </div> <div class="row tab"><!-- THIRD ROW -->
+
+
+
+
+
+
+
+
+
+
+
+
+        </div><!--MAIN ROW CLOSE-->
+
+
+        <!--***************** MAIN CONTENT *******************-->
+<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+
+<script type="text/javascript">
+    var username = 'vasantia';
+    var profileUrl = 'https://api.github.com/users/'+username;
+    var repoUrl = profileUrl+'/repos';
+
+    $.ajax({
+        method: 'GET',
+        url: profileUrl,
+        dataType: 'json'
+    }).done(function(userInfo) {
+        console.log("DONE");
+
+        $('#gitName').replaceWith(userInfo.name);
+        $('#gitLogin').replaceWith(userInfo.login);
+        $('#gitLocation').replaceWith(userInfo.location);
+        $('#gitEmail').replaceWith('vasantia@yahoo.com');
+        $('#gitHtml').replaceWith(userInfo.html_url);
+        $('#profile_image').replaceWith('<img src='+userInfo.avatar_url+' height="215" width="215">');
```

Comment:
A better way to do this might be grabbing the tag and setting it to a variable.  Then you could simply do `tag.attr("src", userInfo.avatar_url)`.